incubator/minecraft/values.yaml

```diff
@@ -0,0 +1,82 @@
+# ref: https://hub.docker.com/r/itzg/minecraft-server/
+Image: itzg/minecraft-server
+ImageTag: latest
+# ref: http://kubernetes.io/docs/user-guide/compute-resources/
+Cpu: 100m
+Memory: 1024Mi
+
+# Most of these map to environment variables. See Minecraft for details:
+# https://hub.docker.com/r/itzg/minecraft-server/
+MinecraftServer:
+  # This must be overridden, since we can't accept this for the user.
+  EULA: "FALSE"
+  # One of: LATEST, SNAPSHOT, or a specific version (ie: "1.7.9").
+  Version: "LATEST"
+  # One of: peaceful, easy, normal, and hard
+  Difficulty: easy
+  # A comma-separated list of player names to whitelist.
+  Whitelist:
+  # A comma-separated list of player names who should be admins.
+  Ops:
+  # A server icon URL for server listings. Auto-scaled and transcoded.
+  Icon:
+  # Max connected players.
+  MaxPlayers: 20
+  # This sets the maximum possible size in blocks, expressed as a radius, that the world border can obtain.
+  MaxWorldSize: 10000
+  # Allows players to travel to the Nether.
+  AllowNether: true
+  # Allows server to announce when a player gets an achievement.
+  AnnouncePlayerAchievements: true
+  # Enables command blocks.
+  EnableCommandBlock: true
+  # If true, players will always join in the default gamemode even if they were previously set to something else.
+  ForceGameMode: false
+  # Defines whether structures (such as villages) will be generated.
+  GenerateStructures: true
+  # If set to true, players will be set to spectator mode if they die.
+  Hardcore: false
+  # The maximum height in which building is allowed.
+  MaxBuildHeight: 256
+  # The maximum number of milliseconds a single tick may take before the server watchdog stops the server with the message. -1 disables this entirely.
+  MaxTickTime: 60000
+  # Determines if animals will be able to spawn.
+  SpawnAnimals: true
+  # Determines if monsters will be spawned.
+  SpawnMonsters: true
+  # Determines if villagers will be spawned.
+  SpawnNPCs: true
+  ViewDistance: 10
+  # Define this if you want a specific map generation seed.
+  LevelSeed:
+  # One of: creative, survival, adventure, spectator
+  GameMode: survival
+  # Message of the Day
+  MOTD: "Welcome to Minecraft on Kubernetes!"
+  # If true, enable player-vs-player damage.
+  PVP: false
+  # One of: DEFAULT, FLAT, LARGEBIOMES, AMPLIFIED, CUSTOMIZED
+  LevelType: DEFAULT
+  # When LevelType == FLAT or CUSTOMIZED, this can be used to further customize map generation.
+  # ref: https://hub.docker.com/r/itzg/minecraft-server/
+  GeneratorSettings:
+  WorldSaveName: world
+  # Check accounts against Minecraft account service.
+  OnlineMode: true
+  # If you adjust this, you may need to adjust Memory above to match.
+  JVMOpts: "-Xmx756M -Xms756M"
+  ServiceType: LoadBalancer
+
+  Rcon:
+    # If you enable this, make SURE to change your Password below.
+    Enabled: false
+    Port: 25575
```

Comment:
I took some time looking around at various charts trying to find a unifying style, but it seems like there's a lot of variation. This was one of the things I wasn't sure about.

I don't feel strongly either way. I guess I like the explicit values since I don't have to stick `default` gotmpl things everywhere, but that may not be a good enough reason.